src/react/utils.js

```diff
@@ -928,6 +913,12 @@ export function createInternalReactElement(
   Create.CreateDataPropertyOrThrow(realm, obj, "props", props);
   Create.CreateDataPropertyOrThrow(realm, obj, "_owner", realm.intrinsics.null);
   obj.makeFinal();
+  // If we're in "rendering" a React component tree, we should have an active reconciler
+  let activeReconciler = realm.react.activeReconciler;
+  let createdDuringReconcilation = activeReconciler !== undefined;
```

Comment:
When they're not created in a component (i.e. in the main body of the app). We need this for the follow up PR to be able to know if it can strip out properties.